sim/web/main.go

```diff
@@ -55,17 +57,28 @@ func runServer(useFS bool, host string, launchBrowser bool, inputReader *bufio.R
 
 	http.HandleFunc("/", func(resp http.ResponseWriter, req *http.Request) {
 		resp.Header().Add("Cache-Control", "no-cache")
+		if strings.HasSuffix(req.URL.Path, "/tbc/") {
+			resp.Write([]byte(`
+				<html><body><a href="/tbc/elemental_shaman">Elemental Shaman Sim</a"><br>
```

Comment:
We don't want people to have to modify this file with every new spec. How about an []proto.Spec in register.go?

Reply:
I saw this as temporary until we had the links between them inside the sims themselves.

Reply:
Oh ok, but I think we only want 1 version of the binary so I'm not sure where else it would launch to.